hazelcast-jet-core/src/main/java/com/hazelcast/jet/core/processor/SourceProcessors.java

```diff
@@ -383,6 +386,24 @@ public static ProcessorMetaSupplier streamFilesP(
                 StreamJmsP.supplier(connectionSupplier, sessionFn, consumerFn, flushFn, projectionFn));
     }
 
+    /**
+     * Returns a supplier of processors for {@link
+     * Sources#jdbc(DistributedSupplier, DistributedFunction,
+     * DistributedBiFunction, DistributedFunction)}.
+     */
+    public static <T> ProcessorMetaSupplier readJdbcP(
+            @Nonnull DistributedSupplier<java.sql.Connection> connectionSupplier,
+            @Nonnull DistributedFunction<java.sql.Connection, Statement> statementFn,
```

Comment:
how come sink works with `PreparedStatement` and source only with `Statement`?

Reply:
`PreparedStatement` extends `Statement`, I'll use `Statement` for both source and sink

Reply:
I think it makes sense how it was. `PreparedStatement` extends `Statement` and allows it to bind variables. For source, we don't need to bind variables, because the statement is executed only once.